src/payment/strategies/amazon-pay/amazon-pay-window.ts

```diff
@@ -1,4 +1,7 @@
+import InternalCheckoutSelectors from '../../../checkout/internal-checkout-selectors';
```

Comment:
`import { InternalCheckoutSelectors } from '../checkout';`

Reply:
does not apply for this case